.github/workflows/ci_cd.yml

```diff
@@ -0,0 +1,76 @@
+name: CI/CD
```

Comment:
Let call it `Zulip Desktop CI` or similar?

Reply:
Zulip Desktop CI is perfect for this.